tools/sdg/topics/core_topics.py

```diff
@@ -48,20 +48,34 @@
 _UNDATA_NL_DESCRIPTIONS_FILE = os.path.join(_TMP_DIR,
                                             'undata_nl_descriptions.csv')
 
-API_ROOT = "https://autopush.api.datacommons.org"
+_UNDATA_ILO_ROOT = "dc/g/Custom_UN"
+_UNDATA_ILO_TOPIC_JSON = '../../../server/config/nl_page/undata_ilo_topic_cache.json'
+_UNDATA_ILO_NON_COUNTRY_VARS = '../../../server/config/nl_page/undata_ilo_non_country_vars.json'
+_UNDATA_ILO_MCF_PATH = os.path.join(_TMP_DIR, 'custom_topics_undata_ilo.mcf')
+_UNDATA_ILO_VARIABLES_FILE = 'undata_ilo_variable_grouping.csv'
+_UNDATA_ILO_SERIES_TOPICS_FILE = 'un_ilo_series_topics.csv'
+_UNDATA_ILO_NL_DESCRIPTIONS_FILE = os.path.join(
+    _TMP_DIR, 'undata_ilo_nl_descriptions.csv')
+
+REMOVE_SVG_PREFIX = "Custom_"
+
+API_ROOT = "https://staging.unsdg.datacommons.org"
```

Comment:
Maybe this too should not be hardcoded, but passed via flag?  Deferring to later is OK

Reply:
+1 added a comment.